dbms/src/Core/Field.h

```diff
@@ -152,6 +152,27 @@ class Field
         return *this;
     }
 
+    Field & operator+= (const Field & rhs)
```

Comment:
Better to use `FieldVisitorSum` directly inside `AggregateFunctionSumMap`. You can move it out of `SummingSortedBlockInputStream.cpp` to `FieldVisitors.h` or a separate header file.

Reply:
Fixed, moved `FieldVisitorSum` to `FieldVisitors.h`, though not sure if that more efficient than using overloaded operator `+=`.